<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminDashboard</class>
 <widget class="QDialog" name="AdminDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="nav_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>51</width>
     <height>768</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(127, 130, 181); 
border: none;</string>
   </property>
   <widget class="QPushButton" name="logout_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
 background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="admin_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../icons/admin.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="dashboard_widget" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>561</width>
     <height>151</height>
    </rect>
   </property>
   <widget class="QLabel" name="welcome_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
font-size: 20px;
font-weight: 1000;</string>
    </property>
    <property name="text">
     <string>Welcome Back Admin!</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget1" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: dark-grey;
font-size: 15px;</string>
     </property>
     <property name="text">
      <string>Total employees</string>
     </property>
    </widget>
    <widget class="QLabel" name="total_employees_label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
font-size: 15px;</string>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget2" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: dark-grey;
font-size: 15px;</string>
     </property>
     <property name="text">
      <string>New employees</string>
     </property>
    </widget>
    <widget class="QLabel" name="newemployees_label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
font-size: 15px;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>721</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
font-size: 15px;
font-weight: 500;</string>
    </property>
    <property name="text">
     <string>Employee id</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
font-size: 15px;
font-weight: 500;</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
font-size: 15px;
font-weight: 500;</string>
    </property>
    <property name="text">
     <string>Department</string>
    </property>
   </widget>
   <widget class="QListWidget" name="id_listwidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="department_listwidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="name_listwidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>141</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font-size: 17px;
font-weight: 1000;</string>
   </property>
   <property name="text">
    <string>Details of employees</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
font-size: 17px;
font-weight: 1000;</string>
    </property>
    <property name="text">
     <string>Add/Remove employee</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_new_employee_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(127, 130, 181); 
border: none;
color: white;
font-weight: 500;
</string>
    </property>
    <property name="text">
     <string>Add a new employee</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_employee_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(127, 130, 181); 
border: none;
color: white;
font-weight: 500;
</string>
    </property>
    <property name="text">
     <string>Remove an employee</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>560</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
font-size: 17px;
font-weight: 1000;</string>
    </property>
    <property name="text">
     <string>Assign Task</string>
    </property>
   </widget>
   <widget class="QPushButton" name="asign_task_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(127, 130, 181); 
border: none;
color: white;
font-weight: 500;
</string>
    </property>
    <property name="text">
     <string>Assign task</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>650</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
font-size: 17px;
font-weight: 1000;</string>
    </property>
    <property name="text">
     <string>Manage Attendance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="manage_attendance_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(127, 130, 181); 
border: none;
color: white;
font-weight: 500;
</string>
    </property>
    <property name="text">
     <string>Manage attendance</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
